src/elements/content-sidebar/activity-feed/task-new/Task.js

```diff
@@ -96,7 +97,12 @@ class Task extends React.Component<Props, State> {
     };
 
     handleModalClose = () => {
-        this.setState({ isEditing: false });
+        const { onModalClose } = this.props;
+        this.setState({ isEditing: false, modalError: null });
```

Comment:
The initial state for `modalError` is `undefined`. Does `null` here mean something different? If not, we should try to stay consistent and avoid `null`. Also, should the `modalError` be cleared in any other circumstances that may have been previously missed?